monitoring/rid_qualifier/create_flight_record_from_kml.py

```diff
@@ -0,0 +1,358 @@
+                flight_state_vertices.append(point1)
+                break
+            state_vertex = get_vertex_between_points(point1, point2, remaining_flight_distance)
+            if state_vertex:
+                state_vertex = state_vertex.coords[:][0]
+                flight_state_vertices.append(state_vertex)
+                if state_vertex == point2:  # This is the special case when remaining_distance is very close to flight_distance
+                    point2 = next(points, None)
+                    if not point2:
+                        flight_state_vertices.append(point1)
+                        break
+                point1 = state_vertex
+        
+    return flight_state_vertices, flight_state_speeds
+
+
+def get_vertex_between_points(point1, point2, at_distance):
+    """Returns vertex between point1 and point2 at a distance from point1.
+    Args:
+        point1: First vertex having tuple (x,y) co-ordinates.
+        point2: Second vertex having tuple (x,y) co-ordinates.
+        at_distance: A distance at which to locate the vertex on the line joining point1 and point2.
+    Returns:
+        A Point object.
+    """
+    line = LineString([point1, point2])
+    new_point = line.interpolate(at_distance)
+    return new_point
+
+def output_coordinates_to_file(flight_state_coords, filename):
+    """Writes output of state coordinates to a file.
+    Args:
+        flight_state_coords: Unflatten Flight state coordinates.
+        filename: Output file name.
+    """
+    flight_state_vertices_str = '\n'.join(flight_state_coords)
+    with open(f'monitoring/rid_qualifier/test_data/{filename}.txt', 'w') as text_file:
+        text_file.write(flight_state_vertices_str)
+
+def generate_flight_record(
+    state_coordinates, flight_description, operator_location, flight_state_speeds):
+    timestamp = datetime.now()
+    now_isoformat = timestamp.isoformat()
+
+    flight_telemetry: List[List[RIDAircraftState]] = []
+    for coordinates, speed in zip(state_coordinates, flight_state_speeds):
+        timestamp = timestamp + timedelta(0, STATE_INCREMENT_SECONDS)
+        timestamp_isoformat = timestamp.isoformat()
+        aircraft_position = RIDAircraftPosition(
+            lng=coordinates[0],
+            lat=coordinates[1],
+            alt=coordinates[2],
+            accuracy_h=flight_description.get('accuracy_h'),
+            accuracy_v=flight_description.get('accuracy_v'),
+            # TODO: need to check value for extrapolated
+            extrapolated=False,
+            )
+        # TODO: RIDHeight values ?
+        aircraft_height = RIDHeight(distance=ALTITUDE_AGL, reference="TakeoffLocation")
+        rid_aircraft_state = RIDAircraftState(
+            timestamp=timestamp_isoformat,
+            # TODO: operational_status ?
+            operational_status="Airborne",
+            position=aircraft_position,
+            height=aircraft_height,
+            # TODO: track ?
+            track=198.57516619987473,
```

Comment:
It's fine to leave this as a TODO for this PR, but let's make sure it's obvious this isn't a real value -- so, let's use, e.g., 99999 instead of this value.

Reply:
done.